docs/en/observability/splunk-get-started.asciidoc

```diff
@@ -1,9 +1,7 @@
 [[splunk-get-started]]
 = Get started with data from Splunk
```

Comment:
```suggestion
= Migrate data from Splunk
```

Reply:
@colleenmcginnis Maybe we can just sidestep the question of what we mean by getting started by renaming this topic, which is kind of an outlier. Unfortunately the topic relies on users knowing some Elastic concepts up front, but maybe we can make the topic better later on.